app/views/index.blade.php

```diff
@@ -26,7 +26,51 @@
     @endif --}}
 
     <h1>{{ Lang::get('cachet.past_incidents') }}</h1>
+
+    <?php
+        $today = Carbon::now();
+        $startDate = Carbon::now();
+
+        // Check if we have another starting point
+        if (Input::has('start_date')) {
```

Comment:
I think @joecohens was just editing what we had originally. But yes, all of this should be within the controller.